test/setup.js

```diff
@@ -16,7 +16,7 @@
 import os from 'os';
 import 'babel-polyfill';
 import chai, { Assertion } from 'chai';
-import 'chai/register-should';
+import 'chai/register-expect';
```

Comment:
I'm pretty sure this line is specific to `should` and can be removed.

Reply:
[https://github.com/chaijs/chai](https://github.com/chaijs/chai)
I think what this does is register `expect` to `global.expect`, and I think we need it